#123 Remove auto updates for core

kirstyburgoine opened the pull request:
Uses `auto_update_core` filter

Fixes #123

roborourke commented:
@kirstyburgoine I can't seem to find where this filter is used in WP, searching the codebase just takes me to the `WP_AUTO_UPDATE_CORE` constant

kirstyburgoine commented:
@roborourke Sorry, this should have had WIP on it still. I found the filter here: https://wordpress.org/support/article/configuring-automatic-background-updates/#core-updates-via-filter but I haven't tested locally yet

roborourke commented:
Hm, I think those docs are wrong or maybe for a future update :/ I can see the specific update type filters but not that global one. The constant is definitely present though

roborourke commented:
Lets define this as false too `AUTOMATIC_UPDATER_DISABLED`

kirstyburgoine commented:
@roborourke I've updated this to use `AUTOMATIC_UPDATER_DISABLED` now, I haven't also added `WP_AUTO_UPDATE_CORE` as per the original ticket because according to this doc: https://wordpress.org/support/article/configuring-automatic-background-updates/ `AUTOMATIC_UPDATER_DISABLED` will automatically disable all updates including core updates so it doesn't seem to be needed

roborourke commented on inc/remove_updates/namespace.php:
```diff
@@ -8,6 +8,7 @@
 function bootstrap() {
 	add_action( 'admin_init', __NAMESPACE__ . '\\remove_maybe_update_checks', 1 );
 	add_action( 'admin_init', __NAMESPACE__ . '\\remove_update_nag' );
+	add_action( 'admin_init', __NAMESPACE__ . '\\remove_auto_updates' );
```
Last thing to do here - I don't think `admin_init` will always be fired in the context of upgrades / installs so you can call the function directly from here, it's fine as we're just defining a constant so earlier the better.

roborourke commented on inc/remove_updates/namespace.php:
```diff
@@ -38,6 +39,14 @@ function remove_update_nag() {
 
+/**
+ * Remove all automatic updates.
+ *
```
```suggestion
```
